doc/articles/uno-development/docfx.md

```diff
@@ -123,5 +123,5 @@ To run the script on Windows:
 
 1. Make sure `crosstargeting_override.props` is not defining UnoTargetFrameworkOverride
 2. Open a Developer Command Prompt for Visual Studio (2019 or 2022)
-3. Go the uno\build folder (not the uno\src\build folder)
+3. Go to the uno\build folder (not the uno\src\build folder)
```

Comment:
Missing the same change for [doc/articles/uno-development/building-uno-ui.md](https://github.com/unoplatform/uno/pull/15122/files/c4c83ae35453d8d047d165b773c0afbffc551f69#diff-ac9a5a0dccd8fcb364c2b7c7dd9c9fdeff26352b14ba44f26b05562d035ba5ab):
 https://github.com/unoplatform/uno/pull/15122/files#discussion_r1459952331